### DISCUSSION TITLE
Daily 18/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#318] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve con la instalación de django que me llevo bastante tiempo, así que no pude leer lo de la plataforma ni hacer las correcciones a la página de perfil. Pero pude ver el video de la clase donde se explica como trasladar la base de datos a django. Por la noche tuve la clase de Inglés y luego cree una de las clases para la base de datos.
Hoy voy a estar creando las tareas que faltan. Voy a crear alguna clase mas y trataré de terminar con la página de Perfil. Por la tarde tengo clase de Programación del Ispc.

**Comment by BraianTroncoso** [comment]
Ayer tuve clases de python, y de java(la de java al final no la pude realizar por completo), quizas si hoy puedo la hago.
Anoche tambien estuve con el tema de consumir la API externa, recién hoy pude realizarlo. También vi la clase compartida del drive sobre Django.
Hoy tengo clases de Programación con el ISPC, si puedo me asigno alguna tarea mas.

**Comment by Brian-Almada** [comment]
Ayer terminé con unas tareas que me quedaban pendientes y me puse a ver lo de Django. Me costó bastante agarrarle la mano así que hoy me puse a revisar el video de nuevo y tratar de entender un poco más. Ahora con clase de Ingles.

**Comment by rpgrca** [comment]
Hice varias modificaciones a los modelos, también estuve transformando a los servicios de angular para que usen un json, todavía me queda unir todo el dashboard en uno solo. Y a revisar todo el sistema para ver si no falta nada.
